document/CoreFormat.mdk

```diff
@@ -340,9 +340,51 @@ run: {
 ```
 ~
 
+### `logicalLocations` property { #run-logicalLocations }
+
+A `run` object should contain a property named `logicalLocations` whose value is a dictionary,
+each of whose properties represents the logical location of one or more results produced in the course of the run.
+
+With one exception described in &sect;[#location-logicalLocationKey], each property name in the `logicalLocations` object shall be
+a string representing the logical location where the result was produced, in a format consistent with
+the programming language in which the programmatic construct specified by that logical location was expressed.
+We refer to this string as a "fully qualified logical name".
+See &sect;[#location-fullyQualifiedLogicalName] for examples.
```

Comment:
We don't refer to this string as a fully qualified name, a logical location object contains a property 'name', which is an identifier for some portion of the logical name. fullyQualifiedLogicalName lives on the location object.